.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,13 @@
+name: Pull Request validation Action workflow.
+on: [pull_request]
+jobs:
+  pr_validation:
+    runs-on: ubuntu-latest
+    name: A job to run test.
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Yes, it's needed because without it the `lint:changes`(which is doing some sort of commit comparisons, so it needs the branch history) script fails every time as you can see [here](https://github.com/instructure/instructure-ui/runs/1533881141?check_suite_focus=true#step:3:1787) in the logs.

Reply:
Also the run time of the github action worklow is the same as the old travis pipeline, so I think travis does this by default.

Reply:
ok, thanks!